.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false # don't cancel all jobs if some of them failed
       matrix:
-        node: ["10", "12"]
+        node: [16.x]
```

Comment:
We should test at more versions, do we only want to support 16? https://nodejs.org/en/about/releases/